### PR TITLE
feature(resetlocalschema): run 'resetlocalschema'

### PR DESCRIPTION
**Pull request by juliayakovlev**
https://github.com/scylladb/scylla/commit/d3a5814f4fd0ef80f3dbfde4095eb11feb05745a
New nemesis: perform schema recalculation by running 'nodetool resetlocalschema'

'resetlocalschema' option was enabled by the 
https://github.com/scylladb/scylla-tools-java/commit/142f517a232f7536eaa95ce1a8f6dd3435f9f990

But it's still not in the ami. This is the reason that this PR is not ready for review - I need to test it

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/nemesis.py [hunk 1]
```diff
@@ -423,6 +423,33 @@ def disrupt_restart_then_repair_node(self):  # pylint: disable=invalid-name
         self.target_node.wait_jmx_up()
         self.repair_nodetool_repair()
 
+    def disrupt_resetlocalschema(self):  # pylint: disable=invalid-name
+        self._set_current_disruption('ResetLocalSchema %s' % self.target_node)
+
+        # TODO: remove the validation below when resetlocalschema option will be available in the 4.5-dev
```
Comment (roydahan) [inline sdcm/nemesis.py:429]: leave this validation.
It'll be needed in enterprise versions.

Comment (roydahan) [reply sdcm/nemesis.py:429]: (remove the TODO)

**Review by roydahan — APPROVED** [review]
LGTM. just minor comment about the TODO comment.

File: sdcm/nemesis.py [hunk 2]
```diff
@@ -423,6 +423,32 @@ def disrupt_restart_then_repair_node(self):  # pylint: disable=invalid-name
...
+
+        rlocal_schema_res = self.target_node.follow_system_log(patterns=["schema_tables - Schema version changed to"])
+        self.target_node.run_nodetool("resetlocalschema")
+        assert list(rlocal_schema_res), "Schema version has not been recalculated"
+
+        # Waiting 60 sec: this time is defined by Tomasz
+        self.log.debug("Sleep for 60 sec: the other nodes should pull new version")
+        time.sleep(60)
+
+        # Check schema version on the nodes - expected the same version on the all nodes
```
Comment (bentsi) [inline sdcm/nemesis.py:441]: no need to check that, we run health check validator that verifies that schema is the same on all nodes

**Review by bentsi — CHANGES_REQUESTED** [review]
please address comments

**Review by bentsi — APPROVED** [review]

**Pull request merged**